provider/database_db_homes_resource_test.go

```diff
@@ -0,0 +1,216 @@
+// Copyright (c) 2017, Oracle and/or its affiliates. All rights reserved.
+
+package provider
+
+import (
+	"testing"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/oracle/bmcs-go-sdk"
+	"github.com/stretchr/testify/suite"
+)
+
+type ResourceDBHomeTestSuite struct {
+	suite.Suite
+	Client       *baremetal.Client
+	Provider     terraform.ResourceProvider
+	Providers    map[string]terraform.ResourceProvider
+	TimeCreated  baremetal.Time
+	Config       string
+	ResourceName string
+	Res          *baremetal.DBHome
+	DeletedRes   *baremetal.DBHome
+}
+
+func (s *ResourceDBHomeTestSuite) SetupTest() {
+	s.Client = testAccClient
+	s.Provider = testAccProvider
+	s.Providers = testAccProviders
+
+	s.TimeCreated = baremetal.Time{Time: time.Now()}
+
+	s.Config = `
+resource "oci_database_db_home" "testDBHome" {
+	#Required
+	database {
+		#Required
+		admin_password = "${var.database_admin_password}"
+		db_name = "${var.database_db_name}"
+	}
+	db_system_id = "${var.db_system_id}"
+	db_version = "${var.db_version}"
+}
+
+	`
+
+	s.Config += testProviderConfig()
+
+	s.ResourceName = "oci_database_db_home.t"
+
+}
+
+func (s *ResourceDBHomeTestSuite) TestCreateResourceDBHome() {
+
+	resource.UnitTest(s.T(), resource.TestCase{
+		Providers: s.Providers,
+		Steps: []resource.TestStep{
+			{
+				ImportState:       true,
+				ImportStateVerify: true,
+				Config:            s.Config,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(s.ResourceName, "display_name", "display_name"),
+					resource.TestCheckResourceAttrSet(s.ResourceName, "id"),
+				),
+			},
+		},
+	})
+}
+
+func (s ResourceDBHomeTestSuite) TestUpdateDBHomeDBVersion() {
```

Comment:
need to remove all the update stuff. I don't think these tests can pass as written. Need to change all the "newValue" with actual valid values.